modules/load_medical_images.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<p>Maybe should updated to be "Download an example image"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>